data/assets/scene/solarsystem/missions/jwst/trail.asset

```diff
@@ -28,12 +28,12 @@ local JWSTTrailLaunch = {
   Renderable = {
     Type = "RenderableTrailTrajectory",
     Translation = {
-			Type = 'SpiceTranslation',
-			Target = 170, -- JWST
-			Observer = 'EARTH',
-			Frame = 'IAU_EARTH',
-			Kernels = { kernels .. "webb.bsp" }
-		},
+      Type = 'SpiceTranslation',
```

Comment:
I think the spacing before was correct?

Reply:
It looked correct when I viewed it in Visual Studio Code before, and it looks the same now too with this fix. I think there were some invisible tabs or something there before that are now removed. It looks strange here but when I open the file it looks exactly the same as before. No indentation has actually changed.

Reply:
Yep, there were tabs: 
![image](https://user-images.githubusercontent.com/8808894/187916384-15d78806-524d-4fe4-9d92-3afd13af8a94.png)